include/widgets-options-filters.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class PLL_Widgets_Filters
+ *
+ * Add new options to {@see https://developer.wordpress.org/reference/classes/wp_widget/ WP_Widget} and saves them.
+ */
+class PLL_Widgets_Options_Filters {
+
+	/**
+	 * @var PLL_Model
+	 */
+	public $model;
+
+	/**
+	 * PLL_Widgets_Filters constructor.
+	 *
+	 * @param PLL_Base $polylang
+	 * @return void
+	 *
+	 * @since 3.0 Moved actions from PLL_Admin_Filters.
+	 */
+	public function __construct( $polylang ) {
+		$this->model = $polylang->model;
+
+		add_action( 'in_widget_form', array( $this, 'in_widget_form' ), 10, 3 );
+		add_filter( 'widget_update_callback', array( $this, 'widget_update_callback' ), 10, 4 );
+	}
+
+	/**
+	 * @param WP_Widget $widget
+	 * @param null      $return
+	 * @param array     $instance
+	 * @return void
+	 *
+	 * @since 3.0 Moved PLL_Admin_Filters
+	 */
```

Comment:
Same as above for `@since`. Also please add the removed short description for parameters.